documentation/Preferences.md

```diff
@@ -22,14 +22,18 @@ The available properties are as follows (almost  are optional, as the system has
   The default is no configuration -- no emails will be sent.  
   Note: the **password** is not stored here, for security reasons. It must be passed in to the server as and environment variable: `SMTP_PASSWORD`. This should be in an `.env` file for development, or as part of the server start-up command for a live server.
 - **`systemManagerPermissionName`**: The "system manager" is a special permission that has certain system management rights (but not as extensive as "Admin"). Any existing permission name can be used for this special permission, in which case it should be specified here. (Default: `systemManager`)
+- **`managerCanEditLookupTables`**: If `true`, then users with the above management permission are also allowed to view/edit lookup tables. (Default: `true`)
 - **`previewDocsMinKeepTime`**: Documents generated as part of the Preview functionality will be periodically cleaned up, as they have no lasting use. It should be a Postgres duration string. (Default: "2 hours").
-- **`previewDocsCleanupSchedule`**: The schedule for cleaning up (deleting) the preview documents, as per the node-schedule syntax above. (Default: daily at 1:05am UTC)
+- **`fileCleanupSchedule`**: The schedule for cleaning up (deleting) and missing files, orphan file database records, and files marked as "to be deleted" (e.g Preview docs), as per the node-schedule syntax above. (Default: daily at 1:05am UTC)
```

Comment:
Have made the names of these schedules more consistent and accurate:
- actionSchedule
- fileCleanupSchedule
- backupSchedule
- archiveSchedule

This would have an effect on the backwards compatibility of existing systems. But considering we'll mostly just use the defaults and the fact that the timezone was messing up the schedules anyway, this won't matter.